### PR TITLE
recommit

### PR DESCRIPTION
**Pull request by SergeKrier**

### CODE REVIEW

File: pkg/bgp/extended-community.go [hunk 1]
```diff
@@ -48,14 +48,23 @@ func (ext *ExtCommunity) String() string {
 		prefix = "sas="
 	case 0x0a:
 		prefix = "l2i="
+	case 0x0b:
+		prefix = "color="
```
Comment (sbezverk) [inline pkg/bgp/extended-community.go:52]: Just a question. Is there any convention/abbreviation which should be used for identification of type of communities?

File: pkg/bgp/extended-community.go [hunk 2]
```diff
@@ -48,14 +48,23 @@ func (ext *ExtCommunity) String() string {
...
 	}
+
+
```
Comment (sbezverk) [inline pkg/bgp/extended-community.go]: Please remove blank lines.

File: pkg/bgp/extended-community.go (hunk 1 above)
Comment (SergeKrier) [reply pkg/bgp/extended-community.go:52]: no official ones that I know, but would be good practice to use those displayed in XR as ref

File: pkg/bgp/extended-community.go [hunk 3]
```diff
@@ -109,6 +118,12 @@ func makeExtCommunity(b []byte) (*ExtCommunity, error) {
 		ext.SubType = &st
 		l = 6
 		p++
+	case 3: 
+		st := uint8(b[p])
+                ext.SubType = &st
+		l = 6
+		p+=3
```
Comment (sbezverk) [inline pkg/bgp/extended-community.go]: is only 4 last bytes used for color community?

Comment (SergeKrier) [reply pkg/bgp/extended-community.go]: correct, first 2 bytes are reserved, although following draft will use 2 bits (co-flags) : ietf-spring-segment-routing-policy ... to be added later

**Pull request merged**